appium/tests/specs/mock/inbox/CheckSecurityWarningForSuspiciousEmail.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { MockApi } from 'api-mocks/mock';
+import { MockApiConfig } from 'api-mocks/mock-config';
+import { SplashScreen, SetupKeyScreen, MailFolderScreen, EmailScreen } from '../../../screenobjects/all-screens';
+
+describe('INBOX: ', () => {
+  it('user is able to view correct signature badge for all cases', async () => {
+    const mockApi = new MockApi();
+
+    const subject = 'Test Spoofed email by Mart';
+    mockApi.fesConfig = MockApiConfig.defaultEnterpriseFesConfiguration;
+    mockApi.ekmConfig = MockApiConfig.defaultEnterpriseEkmConfiguration;
+    mockApi.addGoogleAccount('e2e.enterprise.test@flowcrypt.com', {
+      messages: [subject],
+    });
+
+    await mockApi.withMockedApis(async () => {
+      await SplashScreen.mockLogin();
+      await SetupKeyScreen.setPassPhrase();
+      await MailFolderScreen.checkInboxScreen();
+
+      await MailFolderScreen.clickOnEmailBySubject(subject);
+
+      await EmailScreen.checkSecurityWarningBlock();
```

Comment:
let's move this test to `CheckMessageProcessingErrors` test, which includes other message processing errors. having all such cases in 1 test should make test execution faster, than having a separate test.

Reply:
Okay